### ISSUE TITLE
Error message on function-order rule is misleading

### ISSUE THREAD

**Issue by elopio** [duaraghav8/Solium#180]
Let's say I have this contract:
```
contract TestContract {                                                                                                                                                                 
        function TestContract() {}                                                                                                                                                            
        function testPublicFunction() public {}                                                                                                                                               
        function testExternalFunction() external {}                                                                                                                                           
}
```
If I run solium there, it should fail because external functions should go before public functions. solium indeed fails here, however, the error message says:

`external function(s) should appear after fallback function(s).`

This is very confusing, because my contract has no fallback functions. I thought solium was parsing a fallback function by mistake somehow, it never occurred to me that the error was pointing on the wrong direction.

**Comment by elopio** [comment]
It seems very hard to craft the right message that will fit all the possible contracts, which are likely to not have all the types of function. Maybe, an easier solution would be to just say something like:
`The order should be: constructors, fallback, external, public, internal and private.`

**Comment by duaraghav8** [comment]
Solved in 1.1.6

**Comment by elopio** [comment]
Thank you!
